src/bap_mc/bap_mc.ml

```diff
@@ -170,6 +201,22 @@ module Cmdline = struct
     Arg.(value & opt_all ~vopt:`bil (enum formats) [] &
          info ["show-bil"] ~doc)
 
+  let addr_offset =
+    let doc = sprintf
+        "Specify a pretend offset, as though the instructions\
+	 occur at a certain address, and accordingly \
+	 interpreted. Bases can be hex or decimal " in
+    Arg.(value & opt ~vopt:0 (int) 0 &
+	 info ["offset"] ~doc)    
+
+  let stop_at =
+    let doc = sprintf
+		"Specify a number of instructions to disassemble.\
+		 Good for ensuring that only one instruction is ever\
+		 lifted or disassembled from a byte blob. 0 connotes all" in
+    Arg.(value & opt ~vopt:0 (int) 0 &
+	   info ["ins_limit"] ~doc)
```

Comment:
it is not clear whether it supposed to be minimum or maximum number of instructions? Also, command line arguments usually use hyphen `-` instead of underscores. So better name, would be `max-insns` or `min-insns`.